packages/integration-tests/tests/wellknown.test.ts

```diff
@@ -0,0 +1,40 @@
+import { adminConsoleApplicationId } from '@logto/schemas/lib/seeds';
+import { assert } from '@silverhand/essentials';
+
+import { getWellKnownSignInExperience } from '@/api';
+import MockClient from '@/client';
+import { adminConsoleRedirectUri } from '@/constants';
+
+describe('wellknown api', () => {
+  it('get /.well-known/sign-in-exp for AC', async () => {
+    const client = new MockClient({ appId: adminConsoleApplicationId });
+    await client.initSession(adminConsoleRedirectUri);
+
+    assert(client.interactionCookie, new Error('Session not found'));
```

Comment:
just curious why not use `expect(…).toMatchError(…)`?

Reply:
not for test use. interactionCookie should not be empty here, just use it as a type guard.